unified-codes/apps/web/src/App.tsx

```diff
@@ -9,7 +9,7 @@ import { Container, Grid, AlertBar } from '@unified-codes/ui';
 import { IAlert } from '@unified-codes/data';
 
 import { AlertActions } from './actions';
-import { Explorer, Footer, Header, Login } from './components';
+import { Explorer, Footer, Header, Login, PropertyBrowser } from './components';
```

Comment:
Indeed! Thanks for the suggestions. I agree, `Details` would probably work to match the URL. The reason I didn't go with Entity in the name was due to potential confusion with the EntityBrowser and EntityTable components 😖, but since they are in a different folder maybe they would have also been ok 😄